code/_core/obj/structure/interactive/drop_pod.dm

```diff
@@ -116,18 +116,17 @@ var/global/list/turf/drop_pod_turfs = list() //Drop pods that need to respawn.
 			if(!desired_loc)
 				caller.to_chat(span("warning","Invalid drop location: No drop location selected."))
 				return FALSE
-			for(var/d in DIRECTIONS_ALL + 0x0)
-				var/turf/T = get_step(desired_loc,d)
-				if(T.has_dense_atom)
-					caller.to_chat(span("warning","Invalid drop location: Target area is obstructed."))
-					return FALSE
-				if(!T.is_safe() || !T.is_safe_move())
-					caller.to_chat(span("warning","Invalid drop location: Unsafe area."))
-					return FALSE
-				var/area/A = T.loc
-				if(A.interior)
-					caller.to_chat(span("warning","Invalid drop location: Target area is obstructed."))
-					return FALSE
+			var/turf/T = get_step(desired_loc,SOUTH)
+			if(T.has_dense_atom)
+				caller.to_chat(span("warning","Invalid drop location: Target area is obstructed."))
+				return FALSE
+			if(!T.is_safe() || !T.is_safe_move())
+				caller.to_chat(span("warning","Invalid drop location: Unsafe area."))
+				return FALSE
+			var/area/A = T.loc
+			if(A.interior)
+				caller.to_chat(span("warning","Invalid drop location: Target area is obstructed."))
+				return FALSE
```

Comment:
should probably revert the changes here as this is old drop pod turf checking code

Reply:
This is actually preferred from my experience, as many players would come in and ask if "drop pods were bugged" as it would often seemingly return false positives. The new delete code should fix the issue of it blocking off key areas.